test/e2e/run_test.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# NOTE: Kmesh e2e test framework is heavily inspired by istio integration 
+# framework (https://github.com/istio/istio/tree/master/tests/integration),
+# both in architecture and code.
+
+# Exit immediately for non zero status
+set -e
+
+DEFAULT_KIND_IMAGE="kindest/node:v1.30.0@sha256:047357ac0cfea04663786a612ba1eaba9702bef25227a794b52890dd8bcd692e"
```

Comment:
If you limit the kind/node image version, you will also have a limit on the kind version.
I have another option：Check for kind, then create cluster without specifying image.